ci/d.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+name: D
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: dlang-community/setup-dlang@7c3e57bdc1ff2d8994f00e61b3ef400e67d2d7ac
```

Comment:
Shouldn't we use master directly if this is the template for users and they have to trust setup-dlang either way? (and if they are really worried they can always lock the version).
The way I see it this might hurt the standard user as he won't be able to receive any bugfix to the build pipeline.

Reply:
Unfortunately that's the way the guidelines want it to be: https://github.com/actions/starter-workflows/blob/0170fa5239991d2141e572a1b334ea577ea4f060/.github/pull_request_template.md

Reply:
[]()